.github/workflows/gradle.yaml

```diff
@@ -0,0 +1,33 @@
+name: Java CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - feature/*
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ inputs.platform }} @ ${{ github.event.pull_request.number || github.sha }}'
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@ccb4328a959376b642e027874838f60f8e596de3
```

Comment:
nit: let's add a comment here specifying gradle/wrapp-validation-action version being used.

Reply:
kaido, which version is this?

Reply:
Just add a comment specifying what version is being used (i.e. v1.0.6), like we do here: https://github.com/youtube/cobalt/blob/main/.github/actions/docker/action.yaml#L24

Reply:
added the comment for the version